Lecture2/Task5/Task5L2.java

```diff
@@ -0,0 +1,154 @@
+import java.io.*;
+import java.util.*;
+
+public class Task5L2 {
+
+    private static final String REGEX_NAME_LASTNAME = "[a-zA-Z]+";
+    private static final String MESSAGE_AGAIN = "Again? Really? Try something else :)";
+
+    private static BufferedReader reader = new BufferedReader ( new InputStreamReader ( System.in ) );
+
+    public static
+    void main ( String[] args ) throws Exception {
+        System.out.println ( "Let's start the game!!!" );
+        Console console = System.console ( );
+        String name = null;
```

Comment:
Please, read task carefully. You need to guess your own name. No need to get firstName and lastName from the user's input

Reply:
just for improvement, not interesting to play with the same words all time :)